src/components/AnchorScrollNavbar/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+interface AnchorScrollNavbarProps {
+    className?: string
+}
+
+export const AnchorScrollNavbar = ({ className = '' }: AnchorScrollNavbarProps) => {
+    const baseClasses =
+        'rounded bg-gray-200 bg-opacity-10 w-full max-w-3xl mx-auto flex justify-between items-stetch p-3 sticky top-3'
```

Comment:
I think this needs to be `relative` and not `sticky`? So it doesn't move with the page. Unless we want that, then it just needs to be darker/not transparent